operator/cmd/mesh/manifest.go

```diff
@@ -29,19 +29,24 @@ func ManifestCmd(ctx cli.Context) *cobra.Command {
 	}
 
 	mgcArgs := &ManifestGenerateArgs{}
+	mtcArgs := &ManifestTranslateArgs{}
 
 	args := &RootArgs{}
 
 	mgc := ManifestGenerateCmd(ctx, args, mgcArgs)
+	mtc := ManifestTranslateCmd(ctx, mtcArgs)
 	ic := InstallCmd(ctx)
 
 	addFlags(mc, args)
 	addFlags(mgc, args)
+	addFlags(mtc, args)
```

Comment:
The `dry-run` flag is added but not used.

Reply:
thanks